<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyCreditWindow</class>
 <widget class="QDialog" name="MyCreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Aktualizuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="myCreditLabel">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight Condensed</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mój harmonogram spłat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="creditNumberLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kredyt:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="creditNumberBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="startDateLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Data rozpoczęcia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startDateEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="financialPeriodLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Okres finansowania: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="financialPeriodEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="investmentValueLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wartość inwestycji: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="investmentValueEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="interestLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Oprocentowanie roczne:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="interestEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="commisionLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Prowizja: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commisionEdit">
        <property name="maximumSize">
         <size>
          <width>1500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="installmentLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Miesięczna rata: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="installmentEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>301</width>
     <height>143</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="nextDateLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Data następnej spłaty:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nextDateEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="interestAmountLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kwota odsetek: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="interestAmountEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="capitalAmountLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kwota kapitału: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="capitalAmountEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="debtBalanceLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Saldo zadłużenia: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="debtBalanceEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
